pkg/grpcutil/grpcutil.go

```diff
@@ -33,6 +33,7 @@ type SecurityConfig struct {
 	// KeyPath is the path of file that contains X509 key in PEM format.
 	KeyPath        string `toml:"key-path" json:"key-path"`
 	ClientCertAuth bool   `toml:"client-cert-auth" json:"client-cert-auth"`
+	AllowedCN      string `toml:"allowed-cn" json:"allowed-cn"`
```

Comment:
Can you change `allowed-cn` to `cert-allowed-cn` like other components?

Reply:
ptal